Source/MDKWhitelistExtractor/MDKWhitelistExtractor.csproj

```diff
@@ -37,6 +37,7 @@
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" />
```

Comment:
@malware-dev same as in https://github.com/malware-dev/MDK-SE/pull/213/files#r523456985